packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -87,6 +88,18 @@ export enum DensityTypes {
   Comfortable = 'comfortable',
 }
 
+/**
+ * Sets the types of the texts in the grid.
+ */
+export interface LocaleText {
+  // Density selector toolbar button text
+  densityText?: string | React.ReactNode;
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/2338e34f760b10c7599d22a58ce41295a9d7c286/types/react/index.d.ts#L232-L237

```suggestion
  densityText?: React.ReactNode;
```